index.d.ts

```diff
@@ -26,14 +26,14 @@ export namespace gzip {
 
   }
 
-  export class UncompressStream {
+  export class UncompressStream extends WriteStream {
 
     constructor(opts?: {
       zlib?: object,
       source: sourceType
     });
 
-    on(event: 'error', cb: Function)
+    on(event: 'error' | string, listener: (...args: any[]) => void): this
```

Comment:
```suggestion
    on(event: string, listener: (...args: any[]) => void): this
```
`'error' | string` is the same as `string`

Reply:
If the two listener functions are the same , **tslint**'s tips will show that 'TSLint: These overloads can be combined into one signature taking `string | 'error'`. (unified-signatures)'
![image](https://user-images.githubusercontent.com/8382136/60325616-50330b00-99ba-11e9-86b5-5836ba616605.png)

```ts
      on(event: string, listener: (...args: any[]) => void): this
      on(event: 'entry', listener: (header: streamHeader, stream: WriteStream, next: () => void) => void): this
      on(event: 'finish', listener: () => void): this
      on(event: 'error', listener: (err: Error) => void): this
```